spec/datadog/ci/ext/environment_spec.rb

```diff
@@ -4,18 +4,58 @@
 require 'datadog/ci/ext/environment'
 
 RSpec.describe Datadog::CI::Ext::Environment do
+  fixture_dir = "#{File.dirname(__FILE__)}/fixtures/"
```

Comment:
Minor: Suggest turning this into a constant

Reply:
It's what I've done at first, but Rubocop is mad at it ("Don't declare constants in a dynamic block").
I think I can likely tune this specific cop to not check spec files, it might make sense.

Reply:
I was going to disable the relevant Rubocop cops for specs, but they are actually quite good at catching bad constant patterns in tests: 
```
spec/datadog/ci/ext/environment_spec.rb:7:3: W: Lint/ConstantDefinitionInBlock: Do not define constants this way within a block.
  FIXTURE_DIR = "#{File.dirname(__FILE__)}/fixtures/"
  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
spec/datadog/ci/ext/environment_spec.rb:7:3: C: RSpec/LeakyConstantDeclaration: Stub constant instead of declaring explicitly.
  FIXTURE_DIR = "#{File.dirname(__FILE__)}/fixtures/"
  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

I've change it to a constant, and ignored Rubocop for this case.

I was thinking if there was a way to improve the affected cops upstreams, but I think that someone could easily declare the same `FIXTURE_DIR` I did here while trying to actually use it in a test case.

Reply:
Yeah, I guess it's a resonable feedback from Rubocop if you don't treat RSpec blocks as anything special.